stemcell_builder/stages/bosh_users/apply.sh

```diff
@@ -18,8 +18,8 @@ run_in_chroot $chroot "
 groupadd --system admin
 useradd -m --comment 'BOSH System User' vcap
 chmod 755 ~vcap
-echo \"vcap:${bosh_users_password}\" | chpasswd
-echo \"root:${bosh_users_password}\" | chpasswd
+echo "vcap:${bosh_users_password}" | chpasswd
```

Comment:
I am wrong. I have tested again with escape character, it seems to be working. I am going to revert this change.